### ISSUE TITLE
Add presets for rsyslog_rsyslog_d_files

### ISSUE THREAD

**Issue by tersmitten** [Oefenweb/ansible-rsyslog#3]
Add presets for `rsyslog_rsyslog_d_files`. For instance:

```
rsyslog_rsyslog_d_presets:
      20-ufw:
        - rule: ':msg,contains,"[UFW "'
          logpath: '/var/log/ufw.log'
```

In the configuration of the role you can do:

```
rsyslog_rsyslog_d_files:
  50-default: "{{ rsyslog_rsyslog_d_presets['50-default'] }}"
  20-ufw: "{{ rsyslog_rsyslog_d_presets['20-ufw'] }}"
```

I think this will make configuration more DRY.

What do you think @mvdriel?

**Comment by mvdriel** [comment]
@tersmitten Yes, good idea!

**Comment by tersmitten** [comment]
Fixed in #4
